package.json

```diff
@@ -58,23 +59,52 @@
     "reselect": "^4.0.0",
     "seamless-immutable": "^7.1.3",
     "styled-components": "^4.1.1",
+    "ts-toolbelt": "^4.8.3",
```

Comment:
what's this for?

Reply:
This package is just a collection of typescript utility types, I'm not sure what is the preferred way to type redux store, so I basically defined type for store (`AppStore`) and use `O.Path` type to get a type in interface pain

Reply:
It is placed in `dependencies` instead of `devDependencies` due to eslint rule that basically disallows imports from dev

Reply:
I think we can avoid using this library, just by defining a named type for strings in the types file. Then in the nested Store definition we can just reuse it.